src/systemctl/systemctl.c

```diff
@@ -85,6 +85,23 @@
 #include "verbs.h"
 #include "virt.h"
 
+/* The init script exit status codes
+   0       program is running or service is OK
+   1       program is dead and /var/run pid file exists
+   2       program is dead and /var/lock lock file exists
+   3       program is not running
+   4       program or service status is unknown
+   5-99    reserved for future LSB use
+   100-149 reserved for distribution use
+   150-199 reserved for application use
+   200-254 reserved
+*/
```

Comment:
oh, well, i suggested it the wrong way round... the values from the top are actually right, but the ones from the bottom aren't...